group/assignment6/move_cars.py

```diff
@@ -0,0 +1,38 @@
+def car_positions(state):
+    """ given state returns array where ith element is the position of ith car """
+    positions = [0] * len(start)
```

Comment:
what does "start" mean? Did you mean "state"?

Reply:
Oops, that was clumsy last-minute copy-paste. FIxed

It actually works now:)